client/src/components/Ticketing/checkout/CheckoutPage.tsx

```diff
@@ -19,9 +19,10 @@ import CompleteOrderForm, {CheckoutFormInfo} from './CompleteOrderForm';
 import {selectDonation} from '../ticketingmanager/donationSlice';
 import {useNavigate} from 'react-router-dom';
 
-// FYI this is ok to stay here; it's the public key so other people can't do anything with it anyway.
-// Replace this with your stripe public key
-const stripePromise = loadStripe(process.env.REACT_APP_PUBLIC_STRIPE_KEY);
+//const pk = `${process.env.PUBLIC_STRIPE_KEY}`;
+const pk = `pk_test_51LarKnLnmVGBW71PEQcPH5MivNcFakrZR5hg3jRtSPIp58ziwNHjFfOVktgxWdMD8OHdteqMWiYh1MWCTOY5QcLH00BS27YNOM`;
+//const pk = `pk_live_51LarKnLnmVGBW71PE5iZvRz1YyozIa1DggXkHwUB6xoVwaEhohEZKsestQwUtx0jAl98f3As5T7wIJ7MGDsvHPik000tLSC9xC`;
+const stripePromise = loadStripe(pk);
```

Comment:
I know it's just a public key, but still would be nice to access as an environment variable. Not a blocker